web/themes/custom/novel/templates/layout/html.html.twig

```diff
@@ -29,12 +29,20 @@
   ]
 %}
 <!DOCTYPE html>
-<html{{ html_attributes }}>
-  <head>
-    <head-placeholder token="{{ placeholder_token }}">
-    <title>{{ head_title|safe_join(' | ') }}</title>
-    <css-placeholder token="{{ placeholder_token }}">
-    <js-placeholder token="{{ placeholder_token }}">
+<html{{html_attributes}}>
+	<head>
+		<head-placeholder token="{{ placeholder_token }}">
+			<title>{{ head_title|safe_join(' | ') }}</title>
+			<link rel="preload" fetchpriority="low" href="/themes/custom/novel/assets/dpl-design-system/fonts/gochi-hand-v19-latin-regular.woff2" as="font" type="font/woff2" crossorigin="anonymous"/>
+			<link rel="preload" fetchpriority="low" href="/themes/custom/novel/assets/dpl-design-system/fonts/lora-v26-latin-italic.woff2" as="font" type="font/woff2" crossorigin="anonymous"/>
+			<link rel="preload" href="/themes/custom/novel/assets/dpl-design-system/fonts/lora-v26-latin-regular.woff2" as="font" type="font/woff2" crossorigin="anonymous"/>
+			<link rel="preload" href="/themes/custom/novel/assets/dpl-design-system/fonts/noto-sans-jp-v42-latin-500.woff2" as="font" type="font/woff2" crossorigin="anonymous"/>
+			<link rel="preload" href="/themes/custom/novel/assets/dpl-design-system/fonts/noto-sans-jp-v42-latin-700.woff2" as="font" type="font/woff2" crossorigin="anonymous"/>
+			<link rel="preload" href="/themes/custom/novel/assets/dpl-design-system/fonts/noto-sans-jp-v42-latin-regular.woff2" as="font" type="font/woff2" crossorigin="anonymous"/>
+			<link rel="preconnect" href="http://cover.dandigbib.org"/>
+			<link rel="preconnect" href="https://res.cloudinary.com"/>
```

Comment:
Consider how this is necessary.

Preconnecting to two of the external APIs that we use is helpful but what about FBI, FBS, Publizon and Material List? Is there a reason they are not included?

One simple approach would be to hardcode them all and hope that others will remember to do the same. Another approach would be to let some of the module(s) which define these urls and add them in a `hook_preprocess_page` implementation.

[We could also move the information to a Link HTTP header](https://web.dev/preconnect-and-dns-prefetch/) and avoid "polluting" the DOM.

Reply:
It seems as if [Drupal has a neat way of attaching `<head>` content *and* HTTP headers for render arrays](https://www.drupal.org/node/2160069).

Reply:
It is to be able to serve images as fast as possible. You are right that it could be used for other services, but then again then we end up with having everything as high priority.
I'll leave a note about why I think it is necessary. But I do not think we should invest more time in this matter. We have already spent a lot of time in getting here. :)